### PR TITLE
scaled up ccd data store on prod01 for upgrade on prod00

### PR DESCRIPTION
**Pull request by darren221980**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-8034

### Change description ###

Scaled up ccd-data-store while on prod 01 while prod 00 is taken offline for aks upgrade.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/cluster-00-overlay/ccd/kustomization.yaml [hunk 1]
```diff
@@ -7,3 +7,7 @@ bases:
 patchesStrategicMerge:
 - ../../../namespaces/ccd/message-publisher/prod-00.yaml
 - ../../../namespaces/ccd/ccd-case-disposer/prod-00.yaml
+- ../../../namespaces/ccd/ccd-definition-store-api/patches/prod-00.yaml
+- ../../../namespaces/ccd/ccd-definition-store-api/patches/prod-01.yaml
```
Comment (ciaranca) [inline k8s/prod/cluster-00-overlay/ccd/kustomization.yaml:11]: Conflict here, prod-01 should be in cluster-01-overlay

File: k8s/prod/cluster-00-overlay/ccd/kustomization.yaml [hunk 2]
```diff
@@ -7,3 +7,7 @@ bases:
...
+- ../../../namespaces/ccd/ccd-data-store-api/patches/prod-00.yaml
+- ../../../namespaces/ccd/ccd-data-store-api/patches/prod-01.yaml
```
Comment (ciaranca) [inline k8s/prod/cluster-00-overlay/ccd/kustomization.yaml:13]: Same here, move to cluster-01-overlay

Comment (darren221980) [reply k8s/prod/cluster-00-overlay/ccd/kustomization.yaml]: now fixed.

**Review by reespozzi — APPROVED** [review]

**Review by reespozzi — CHANGES_REQUESTED** [review]
Failing checks

**Pull request closed without merging**